recipes/ESC50/interpret/train_piq.py

```diff
@@ -597,7 +602,7 @@ def audio_pipeline(wav):
         """Load the signal, and pass it and its length to the corruption class.
         This is done on the CPU in the `collate_fn`."""
 
-        wave_file = data_audio_folder + "/{:}".format(wav)
+        wave_file = data_audio_folder + f"/{:}".format(wav)
```

Comment:
Incorrect f-string? This will cause an error afaict.

Reply:
I believe I've fixed it now, and fixed a merge conflict.